src/fswAlgorithms/attGuidance/prescribedTrans/prescribedTrans.i

```diff
@@ -0,0 +1,43 @@
+/*
+ ISC License
+
+ Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+ */
+%module prescribedTrans
+%{
+   #include "prescribedTrans.h"
+%}
+
+%include "swig_conly_data.i"
+%constant void SelfInit_prescribedTrans(void*, uint64_t);
+%ignore SelfInit_prescribedTrans;
+%constant void Reset_prescribedTrans(void*, uint64_t, uint64_t);
+%ignore Reset_prescribedTrans;
+%constant void Update_prescribedTrans(void*, uint64_t, uint64_t);
+%ignore Update_prescribedTrans;
+
+%include "prescribedTrans.h"
+
+%include "architecture/msgPayloadDefC/prescribedMotionMsgPayload.h"
```

Comment:
This needs to be capitalized to match the file name. This is the reason this branch currently fails to build in the Linux CI job.